### PR TITLE
Support multi-hop upgrade on disconnected clusters

### PR DESCRIPTION
**Pull request by shellyyang1989**
The change:
1. supports multi-hop image mirror
2. adds a multi-hop upgrade job for vsphere-ipi-disconnected

### CODE REVIEW

**Comment by shellyyang1989** [comment]
cc @jianlinliu @jiajliu @jinyunma @liangxia @jhou1 for review and visibility. Thanks!

File: ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__nightly-4.11-upgrade-from-stable-4.10-from-stable-4.9.yaml [hunk 1]
```diff
@@ -62,6 +62,31 @@ tests:
           cpu: 100m
           memory: 200Mi
     - chain: cucushift-upgrade
+- as: vsphere-ipi-disconnected
+  cron: 0 22 */3 * *
```
Comment (liangxia) [inline ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__nightly-4.11-upgrade-from-stable-4.10-from-stable-4.9.yaml:66]: Perhaps we can test this kind of upgrade less often. E.g, once every week

Comment (shellyyang1989) [reply ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__nightly-4.11-upgrade-from-stable-4.10-from-stable-4.9.yaml:66]: sure. let me update. Thanks for the suggestion!

**Comment by shellyyang1989** [comment]
The vsphere image registry was running out of disk space so the test failed. But it passed several times. Please check out https://prow.ci.openshift.org/job-history/gs/origin-ci-test/pr-logs/directory/rehearse-29631-periodic-ci-openshift-verification-tests-master-nightly-4.11-upgrade-from-stable-4.10-from-stable-4.9-vsphere-ipi-disconnected.

@liangxia would you mind taking a look and giving a lgtm if no concern? Thanks!!

**Pull request merged**